pom.xml

```diff
@@ -19,7 +19,7 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<java.version>1.8</java.version>
-		<spring-cloud.version>Edgware.RELEASE</spring-cloud.version>
+		<spring-cloud.version>Edgware.BUILD-SNAPSHOT</spring-cloud.version>
```

Comment:
Should we bump this to "Edgware.SR1" before our 0.0.3 release?

Reply:
Yeah, that would make sense.

Reply:
I can change that on merge